Dockerfile

```diff
@@ -22,22 +22,22 @@ FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4-949.1714662671
 RUN microdnf update -y && rm -rf /var/cache/yum
 ADD cmd/csi-driver/AlmaLinux-Base.repo /etc/yum.repos.d/
 
-RUN microdnf install -y cryptsetup tar procps
+RUN microdnf install -y cryptsetup tar procps psmisc
```

Comment:
Shouldn't we borrow `fuser` from the host? That's what I did.

Reply:
Updated